src/Elzik.Breef.Infrastructure/SummarisationInstructions/HtmlContent.md

```
@@ -1,8 +1,24 @@
+# Task
+
+Summarise the provided web page text.
+
+## Input Structure
+
+The provided text is the main content extracted from a web page.
+
+## Requirements
+
+1. **Overview**: 
+  - Summarise text, including HTML entities.
```

markdownlint-cli2 (0.18.1)

Unordered list indentation
Expected: 0; Actual: 2

(MD007, ul-indent)